deploy/ipi-baremetalds-virt/roles/install-dev/files/config_arbiter_example.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+export IP_STACK="v4"
+export NUM_WORKERS=0
+export ENABLE_ARBITER="true"
+export ARBITER_MEMORY=16384
+export MASTER_MEMORY=32768
+export MASTER_DISK=100
+export NUM_MASTERS=2
+export FEATURE_SET="TechPreviewNoUpgrade"
+
+# This is currently being used to avoid needing ci_token
+# You can read more on this here: https://github.com/openshift-metal3/dev-scripts/blob/3f070cfd36977381a186cadfb44887856d652bed/config_example.sh#L21
+export OPENSHIFT_CI="true"
+
+export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=quay.io/openshift-release-dev/ocp-release:4.19.0-rc.5-multi-x86_64
+export OPENSHIFT_RELEASE_IMAGE=quay.io/openshift-release-dev/ocp-release:4.19.0-rc.5-multi-x86_64 
```

Comment:
Is it worth leaving a comment to explain where to find the latest quay.io URL?
Maybe a link to the public sources at mirror.openshift.com